lib/travis/api/v3/renderer/installation.rb

```diff
@@ -0,0 +1,6 @@
+module Travis::API::V3
+  class Renderer::Installation < ModelRenderer
+    representation(:minimal,  :id, :github_id)
+    representation(:standard, *representations[:minimal], :owner_type, :owner_id)
```

Comment:
Instead of exposing `owner_type` and `owner_id`, this should include `owner` (like repository and other entities). Especially exposing `owner_type` is a leaky abstraction, as it would include the Ruby class name. Also, that way it is possible to fetch the owner explicitly via `include`.